### PR TITLE
Changing the banner to indicate that TLS 1.2 enforcement took place

### PR DESCRIPTION
**Pull request by agr**
Linked directly to a blog post with workarounds for affected systems outlined.

### CODE REVIEW

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml [hunk 1]
```diff
@@ -23,8 +23,7 @@
         <div class="col-sm-12">
             <i class="ms-Icon ms-Icon--Warning" aria-hidden="true"></i>
             <span>
-                NuGet.org will permanently <a href="https://devblogs.microsoft.com/nuget/nuget-org-will-permanently-remove-support-for-tls-1-0-and-1-1-on-june-15th/">
-                remove support</a> for TLS 1.0 and 1.1 on June 15th. Please ensure that your systems use TLS 1.2.
+                NuGet.org had TLS 1.0 and 1.1 disabled. Please, refer to our <a href="https://devblogs.microsoft.com/nuget/deprecating-tls-1-0-and-1-1-on-nuget-org/">blog post</a> if you are having connection issues.
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Shared/Gallery/Header.cshtml:26]: I'd consider removing the comma:

```suggestion
                NuGet.org had TLS 1.0 and 1.1 disabled. Please refer to our <a href="https://devblogs.microsoft.com/nuget/deprecating-tls-1-0-and-1-1-on-nuget-org/">blog post</a> if you are having connection issues.
```

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml [hunk 2]
```diff
@@ -23,8 +23,7 @@
         <div class="col-sm-12">
             <i class="ms-Icon ms-Icon--Warning" aria-hidden="true"></i>
             <span>
-                NuGet.org will permanently <a href="https://devblogs.microsoft.com/nuget/nuget-org-will-permanently-remove-support-for-tls-1-0-and-1-1-on-june-15th/">
-                remove support</a> for TLS 1.0 and 1.1 on June 15th. Please ensure that your systems use TLS 1.2.
+                NuGet.org had TLS 1.0 and 1.1 disabled. Please refer to our <a href="https://devblogs.microsoft.com/nuget/deprecating-tls-1-0-and-1-1-on-nuget-org/">blog post</a> if you are having connection issues.
```
Comment (chgill-MSFT) [inline src/NuGetGallery/Views/Shared/Gallery/Header.cshtml:26]: Minor nitpick: I would reword to "NuGet.org has disabled TLS 1.0 and 1.1...." Otherwise looks good 👍
